### ISSUE TITLE
Acoustic screens and acoustic covers

### ISSUE THREAD

**Issue by AmandineHJ** [Universite-Gustave-Eiffel/NoiseModelling#565]
Hello ! 

I am currently trying to put acoustic screens in my model but it seems not to work. I don't know what the problem is. I have put a thin building in my model (as an acoustic screen, next to the road) and add a column with "G" the absorption of this screen, on the building table. You can find attached my file with the differents tries. I have put G=0 and G=1 and I see no difference in the results. Could you help me with this please ?

[Essais_ecrans2.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/11120300/Essais_ecrans2.zip)

Furthermore, I was wondering if we can put acoustic covers on the roads for example. Can we do that ? And if we can, how do we do that ?

Sincerely,
Amandine

**Comment by pierromond** [comment]
This coefficient is influent only if there are some reflections on the wall. Is this the case ?

> Furthermore, I was wondering if we can put acoustic covers on the roads for example. Can we do that ? And if we can, how do we do that ?
This is the PVMT attribute in the ROADS table (see documentation)

**Comment by AmandineHJ** [comment]
When you mean that this coefficient is influent only if there are some reflexions on the wall, you mean I need to change the "order of relexion" ? But the default value is 1, so shouldn't it be working anyway ? 

![image](https://user-images.githubusercontent.com/129366571/229442226-0ceb6e01-bb92-4405-98f8-60d0e0ddc96b.png)

For acoustic covers I didn't mean the ground cover, but really acoustic covers. They are over the roads, this is not tunnels but really acoustic covers. You can see an example here :
![image](https://user-images.githubusercontent.com/129366571/229441716-c587f448-0353-4261-adb5-8d61da921bed.png)

And do you know more about acoustic semi-covers, how do we modelize them ? Acoustic semi-covers are just like acoustic covers but there is only one part of the roads that is concerned. Here is an example of acoustic semi-covers :

![image](https://user-images.githubusercontent.com/129366571/229441802-abdf4ac7-ef05-45ba-a518-4065c36b528a.png)
![image](https://user-images.githubusercontent.com/129366571/229441926-eb44517c-859e-4fe5-9096-f29bb16b1575.png)

**Comment by AmandineHJ** [comment]
Hello,

Did you have the time to look at my question ?
I thank you in advance.

**Comment by pierromond** [comment]
Yes, I am afraid nobody answers fastly because it is not only "NoiseModelling question"... :) Maybe, you can read the CNOSSOS-EU document to better understand how the propagation is modeled in NoiseModelling/CadnaA/Soundplan, etc. 

I am sorry about this answer but this is the best we can do. If you have any question about the use of NoiseModelling, you should have faster ones ;) and we will be happy to answer.

Kind regards.
